README.md

```diff
@@ -13,16 +7,107 @@ which has separate instructions.
+  production-ready. They contain well-tested features suitable for live
+  environments.
+- **When to Use**: Select one of the latest two supported release branches
+  (e.g., `2.7.x-dev`, `2.6.x-dev`) if you need a stable version of the project
+  for production use, have specific environment constraints, or want to maintain
+  compatibility with particular third-party modules or a specific release.
+- **Note**: Only the latest two release branches are actively supported. Using
+  one of these ensures that you receive updates, security patches, and support
+  aligned with your system's requirements, avoiding potential conflicts and
+  achieving a robust and reliable implementation.
+- **Version Constraints in Supported Release Branches**: Each supported release
+  branch in this repository specifies a version constraint for az_quickstart in composer.json.
+  This constraint ensures that the release branch will work with compatible
+  versions of Quickstart that correspond to the compatible minor release branch,
+  providing flexibility while maintaining alignment with the intended versions.
+
+### Feature or Issue Branches
+
+Feature or issue branches in this repository are typically created to make
+updates to the scaffolding to work with specific branches of
+`az-digital/az_quickstart`. When creating such a branch, it is customary to pin
+this repository's composer.json reference for `az-digital/az_quickstart` to a
+specific issue/feature branch that the changes are intended to work with. This
+ensures that the branch's changes align precisely with a particular state of the
+Quickstart project.
+
+These branches are useful for isolated development, testing of changes, or adding new features to the scaffolding that correspond to developments in the Quickstart project.
+
+**Notes**
+- These branches are not as stable as the main or release branches, so use
+  them for development, experimentation, or testing rather than in production
+  environments.
+- The use of feature or issue branches in this repository is tightly aligned
+  with developments in the az-digital/az_quickstart project, and they should be
+  used in conjunction with the corresponding Quickstart branches.
+
+### Using the `composer create-project` command:
+
+With Composer's `create-project` command, you can quickly scaffold a new project
+using the Arizona Quickstart Scaffolding. Each `az-quickstart-scaffolding`
+branch coincides with an `az_quickstart` branch, ensuring compatibility and
+alignment between the scaffolding and the core project.
+
+
+Here are some examples of how you might use this command with different branch
+specifications:
+
+If using any of the following commands, adjust `my_project_name` to your desired
+project directory name.
+
+1. **Using the Main Branch (`main`)**:
+   ```bash
+   composer create-project az-digital/az-quickstart-scaffolding:dev-main my_project_name --no-interaction --no-dev
```

Comment:
```suggestion
   composer create-project az-digital/az-quickstart-scaffolding:dev-main my_project_name --no-interaction
```

Reply:
This suggestion only makes sense if we change the scaffolding repo to use az-digital/az_quickstart: ~2.8.x